### DISCUSSION TITLE
How to remove trailing slash URLs

### DISCUSSION THREAD

**Discussion by oim37** [wintercms/winter#893] [Q&A]
Hello,

I often use {{ this.page.url }} to work with links and this add "/" slash on the end of urls, my nginx setting make all urls without slash by 301 redirect.

How to remove slash in this option: {{ this.page.url }} 
May be i need change some code.. Thanks

**Comment by damsfx** [comment]
Try to use : `{{  this.page.url | trim('/', 'right') }}`.  

See :  https://twig.symfony.com/doc/2.x/filters/trim.html

**Reply by oim37** [reply]
Thank you. I use {{ this.page.url | replace({'/':''}) }} but your solution better.
